src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/Offender.kt

```diff
@@ -112,17 +120,22 @@ class OffenderBuilder(
   ) = bookingBuilderFactory.builder()
     .let { builder ->
       builder.build(
-        offender = offender,
-        // if you want multiple bookings then create them with latest booking first so it gets seq 1 like in Nomis
-        bookingSequence = bookingSequence ?: (offender.bookings.size + 1),
+        offender = aliasOffender,
+        // if you want multiple bookings then create them with latest booking first so it gets seq 1 like in Nomis,
+        // or override the sequences
+        bookingSequence = bookingSequence ?: nextBookingSequence++,
         agencyLocationCode = agencyLocationId,
         bookingBeginDate = bookingBeginDate,
         active = active,
         inOutStatus = inOutStatus,
         youthAdultCode = youthAdultCode,
         livingUnitId = livingUnitId,
       )
-        .also { offender.bookings += it }
+        .also {
+          println("Created $it for ${it.offender}")
+          aliasOffender.bookings += it
+          rootOffender.getAllBookings()?.add(it)
+        }
```

Comment:
Although the lists are correct when retrieved from the db, this is needed so that they are initialised in the records returned from the dsl functions